indico/modules/events/editing/client/js/components/FileManager/index.jsx

```diff
@@ -151,10 +151,14 @@ function FileType({uploadURL, fileType, uploads}) {
                 </Translate>
               }
               trigger={
-                <Segment>
+                <Message
+                  onDismiss={() => {
+                    dispatch(actions.removeInvalidFilename(fileType.id, invalidFile));
+                  }}
```

Comment:
Hm, not sure about this one but you can also omit `{}` which will be the return value for `onDismis`. I don't think `Message` relies on the return value so it would be purely personal preference to change it.

Reply:
With prettier not wanting it in one line (how I had it originally) I think I personally prefer how it is now :)